test/browser_tests/cucumber-new/features/suites/apps/complaint.feature

```diff
@@ -0,0 +1,102 @@
+Feature: Submit a complaint and Consumer Complaint Database
```

Comment:
This file is based around a way out-of-date version of our Submit a Complaint page. Let's drop it.

Reply:
Removed this feature file.